src/node_crypto.cc

```diff
@@ -243,6 +241,13 @@ static int PasswordCallback(char *buf, int size, int rwflag, void *u) {
 }
 
 
+// This callback is used to avoid the default passphrase callback in OpenSSL
+// which will typically prompt for the passphrase.
```

Comment:
"the prompting is designed for the openssl CLI, but works poorly for Node.js because it involves synchronous interaction with the controlling terminal, something we never want, and use this function to avoid."  <--- Maybe add something like that?